packages/diagrams/backend/sirius-components-collaborative-diagrams/src/main/resources/schema/diagram.graphqls

```diff
@@ -103,6 +111,8 @@ type Edge {
   targetObjectId: String!
   targetObjectKind: String!
   targetObjectLabel: String!
+  modifiers: [ViewModifier!]!
```

Comment:
Modifiers are never used by the frontend (they are not requested by the GraphQL query)